modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,255 @@
+      ?)
+        exit_error "invalid argument: ${OPTARG}"
+        ;;
+    esac
+  done
+  shift $((OPTIND - 1))
+
+  check_requirements || exit 1
+
+  ${CMD} -v "${DATA_DIR}:${MUST_GATHER_VOL}:z" "${PAO_IMG}" "$@" --must-gather-dir-path "${MUST_GATHER_VOL}"
+  echo "" 1>&2
+}
+
+main "$@"
+----
+
+. Add execute permissions for everyone on this script:
++
+[source,terminal]
+----
+$ chmod a+x run-perf-profile-creator.sh
+----
+
+. Optional: Display the `run-perf-profile-creator.sh` command usage:
++
+[source,terminal]
+----
+$ ./run-perf-profile-creator.sh -h
+----
++
+.Expected output
++
+[source,terminal]
+----
+Wrapper usage:
+  run-perf-profile-creator.sh [-h] [-p image][-t path] -- [performance-profile-creator flags]
+
+Options:
+   -h                 help for run-perf-profile-creator.sh
+   -p                 Performance Addon Operator image <1>
+   -t                 path to a must-gather tarball <2>
+
+A tool that automates creation of Performance Profiles
+
+   Usage:
+     performance-profile-creator [flags]
+
+   Flags:
+         --disable-ht                        Disable Hyperthreading
+     -h, --help                              help for performance-profile-creator
+         --info string                       Show cluster information; requires --must-gather-dir-path, ignore the other arguments. [Valid values: log, json] (default "log")
+         --mcp-name string                   MCP name corresponding to the target machines (required)
+         --must-gather-dir-path string       Must gather directory path (default "must-gather")
+         --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
+         --profile-name string               Name of the performance profile to be created (default "performance")
+         --reserved-cpu-count int            Number of reserved CPUs (required)
+         --rt-kernel                         Enable Real Time Kernel (required)
+         --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
+         --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
+         --user-level-networking             Run with User level Networking(DPDK) enabled
+----
++
+[NOTE]
+====
+There two types of arguments:
+
+* Wrapper arguments namely `-h`, `-p` and `-t`
+* PPC arguments
+====
++
+<1> Optional: Specify the Performance Addon Operator image. If not set, the default upstream image is used: `quay.io/openshift-kni/performance-addon-operator:4.8-snapshot`.
+<2> `-t` is a required wrapper script argument and specifies the path to a `must-gather` tarball.
+
+. Run the performance profile creator tool in discovery mode:
++
+[NOTE]
+====
+Discovery mode inspects your cluster using the output from `must-gather`. The output produced includes information on the:
```

Comment:
ending with "the" here doesn't make sense for the second bullet. I'd remove it and then add it to the front of just the first bullet.